nx/lib/nx.ex

```diff
@@ -16464,4 +16464,123 @@ defmodule Nx do
       end)
     end
   end
+
+  @doc """
+  Returns the logarithm of the sum of the exponentials of tensor elements.
+
+  If the `:axes` option is given, it aggregates over
+  the given dimensions, effectively removing them.
+  `axes: [0]` implies aggregating over the highest order
+  dimension and so forth. If the axis is negative, then
+  counts the axis from the back. For example, `axes: [-1]`
+  will always aggregate all rows.
+
+  You may optionally set `:keep_axes` to true, which will
+  retain the rank of the input tensor by setting the reduced
+  axes to size 1.
+
+  ## Examples
+
+      iex> Nx.logsumexp(Nx.tensor([1, 2, 3, 4, 5, 6]))
+      #Nx.Tensor<
+        f32
+        6.456193447113037
+      >
+
+      iex> Nx.logsumexp(Nx.tensor([1, 2, 3, 4, 5, 6]), exp_scaling_factor: 0.5)
```

Comment:
I would describe the option `exp_scaling_factor` in docs

Reply:
Agreed, added a short explanation - let me know what you think. Sorry it took me a while.

Reply:
Looks good :)